src/show/components/ProgressBar/index.js

```diff
@@ -0,0 +1,15 @@
+const ProgressBar = ({ percentage, color }) => {
+  return (
+    <div className=' w-3/4  bg-gray-200 rounded-full dark:bg-gray-700'>
```

Comment:
Remove the dark class, we are not yet implementing this.
Remove blank space after quote.